cub/device/dispatch/dispatch_adjacent_difference.cuh

```diff
@@ -175,8 +194,9 @@ struct DispatchAdjacentDifference : public SelectedPolicy
       , num_items(num_items)
       , difference_op(difference_op)
       , stream(stream)
-      , debug_synchronous(debug_synchronous)
```

Comment:
Per Allison's comment, I've added a logging message in the `CUB_DETAIL_RUNTIME_DEBUG_SYNC_USAGE_LOG`, so I uses `debug_synchronous` in it.

Reply:
I actually looked at this and totally missed it. Thanks a lot

Reply:
Ah, I should have caught this -- ideally the macro would take the variable name as an argument to make it clearer what its inputs are and prevent these sorts of questions.

I'm also a big fan of crafting macros that require a `;` after their usage (`do { ... } while (false)` is handy for that). Makes automatic code formatting easier on some IDEs/editors.

No need to go fix this now since it's a minor implementation detail, just wanted to share some thoughts :)